mm2src/coins/test_coin.rs

```diff
@@ -439,7 +447,7 @@ impl CoinAssocTypes for TestCoin {
     type Sig = TestSig;
     type SigParseError = String;
 
-    fn my_addr(&self) -> &Self::Address { todo!() }
+    async fn my_addr(&self) -> Self::Address { todo!() }
```

Comment:
It wasn't named by me, but I think Artem wanted to distinguish between the 2 since this is supposed to be used in swaps v2 (trading protocol upgrade) context.